tests/group_vars/compute3/vars.yml

```diff
@@ -1,3 +1,11 @@
 ip_address: '127.0.0.3'
 mac_address: "00:00:00:00:aa:c3"
+memtest86_0_path: "http://10.1.1.1/memtest.0"
 os_disks: 'sdc'
+kernel_numa_param: "on rhgb quiet"
+dhcp_kickstart_manage_packages: "anything"
+kickstart_packages: |
+  %packages --nobase
+  @core
```

Comment:
Why are you installing so many packages? Won't the test take a longer time?

Reply:
There's no actual PXE booting happening as part of this testing so nothing is installing or running anaconda with these kickstart configs as input(I suspect kickstart works in that manner, I might be wrong).

The test does a "curl http://url/cgi-bin/boot.py" which is what the PXE boot would do.
Really PXE booting something could be a fun test to add, probably very time-consuming and a great way of learning about docker and its networking.

Reply:
That makes sense.